proposals/0038-16bit-isspecialfloat.md

```diff
@@ -58,6 +60,7 @@ These are the existing Opcodes for the isSpecialFloat DXIL OpClass.
 8        | IsNaN       | Returns true if x is NAN or WNAN, false otherwise.
 9        | IsInf       | Returns true if x is +INF or -INF, false otherwise.
 10       | IsFinite    | Returns true if x is finite, false otherwise.
+11       | IsNormal    | Returns IsNormal
```

Comment:
This description is pretty weak.

```suggestion
11       | IsNormal    | Returns false if x is zero, INF, NAN or subnormal (aka. denorm), true otherwise.
```

Reply:
Funny, I borrowed this description from DXIL.rst.